### PR TITLE
Column deletion warnings

### PR DESCRIPTION
**Pull request by shellymathew98**
- Pop-up added to notify user to remove dependancies.
- Informs user if the column is a part of Primary Key or Index.

### CODE REVIEW

File: ui/src/app/components/object-detail/object-detail.component.ts [hunk 1]
```diff
@@ -346,12 +346,60 @@ export class ObjectDetailComponent implements OnInit {
 
   dropColumn(element: any) {
     let colName = element.get('srcColName').value
-    this.spRowArray.value.forEach((col: IColumnTabData, i: number) => {
-      if (col.srcColName === colName) {
-        this.droppedColumns.push(col)
+    let associatedIndexes = this.getAssociatedIndexs(colName)
+    if (this.checkIfPkColumn(colName) || associatedIndexes.length != 0) {
```
Comment (shreyakhajanchi) [inline ui/src/app/components/object-detail/object-detail.component.ts:350]: this check doesn't seem to work if we try dropping a column after modifying its name - it is allowing it to be dropped even though it has dependencies

File: .github/workflows/integration-tests-against-emulator.yaml [hunk 2]
```diff
@@ -13,147 +13,146 @@
 # limitations under the License.
```
[the remark this answers is not in the document]
Comment (shellymathew98) [reply .github/workflows/integration-tests-against-emulator.yaml:13]: This file is required for the Integration tests to pass. There are only 2 line changes, Other changes are editor formatting changes. Following is the only change.

- uses: google-github-actions/setup-gcloud@v1
              with:
                  version: '410.0.0'

File: ui/src/app/components/object-detail/object-detail.component.ts [hunk 3]
```diff
@@ -346,12 +346,60 @@ export class ObjectDetailComponent implements OnInit {
 
   dropColumn(element: any) {
     let colName = element.get('srcColName').value
-    this.spRowArray.value.forEach((col: IColumnTabData, i: number) => {
-      if (col.srcColName === colName) {
-        this.droppedColumns.push(col)
+    let associatedIndexes = this.getAssociatedIndexs(colName)
```
[the remark this answers is not in the document]
Comment (shellymathew98) [reply ui/src/app/components/object-detail/object-detail.component.ts]: Dist files are added now.

File: ui/src/app/components/object-detail/object-detail.component.ts (hunk 1 above)
Comment (shellymathew98) [reply ui/src/app/components/object-detail/object-detail.component.ts]: Fixed.

File: ui/src/app/components/object-detail/object-detail.component.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (shreyakhajanchi) [reply ui/src/app/components/object-detail/object-detail.component.ts]: getting different files generated after running ng build , lets connect if unable to resolve thsi

**Review by shreyakhajanchi — COMMENTED** [review]
Majorly LGTM, fix lint issue and ng build files

File: ui/src/app/components/object-detail/object-detail.component.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (shellymathew98) [reply ui/src/app/components/object-detail/object-detail.component.ts]: Fixed.

**Review by shreyakhajanchi — APPROVED** [review]
LGTM

**Pull request merged**